marathon_acme/tests/test_fake_marathon.py

```diff
@@ -186,3 +186,55 @@ def test_get_event_subscriptions(self):
         response_json = yield json_content(response)
         self.assertThat(response_json,
                         Equals({'callbackUrls': [callback_url]}))
+
+    @inlineCallbacks
+    def test_post_event_subscriptions(self):
+        """
+        When a callback URL is posted for an event subscription, the subscibe
+        event should be returned and FakeMarathon should now have the callback
+        URL registered.
+        """
+        callback_url = 'http://marathon-acme.marathon.mesos:7000'
+
+        response = yield self.client.request(
+            'POST', '/v2/eventSubscriptions',
+            query={'callbackUrl': callback_url})
+        self.assertThat(response, IsJsonResponseWithCode(200))
+
+        response_json = yield json_content(response)
+        self.assertThat(response_json, MatchesDict({
+            'eventType': Equals('subscribe_event'),
+            'callbackUrl': Equals(callback_url),
+            'clientIp': Is(None),  # FIXME: No clientIp in request
```

Comment:
@jerith: Any idea what we can do here? https://github.com/twisted/twisted/blob/twisted-16.2.0/twisted/web/http.py#L1240-L1250